docs/monorepo.md

```diff
@@ -1,77 +1,323 @@
+
+This will allow you to fetch changes from the main Catalyst project and merge them into your fork as needed.
 
 ```shell copy
-corepack enable pnpm && pnpm install
+git remote add upstream git@github.com:bigcommerce/catalyst.git
 ```
 
-3. Set up environment variables by running:
+To confirm, run the following command in your terminal:
 
 ```shell copy
-cp .env.example .env.local
+git remote -v
+```
+
+You should see output similar to:
+
+```shell
+origin          git@github.com:<YOUR_GITHUB_USERNAME>/<YOUR_FORK_NAME>.git (fetch)
+origin          git@github.com:<YOUR_GITHUB_USERNAME>/<YOUR_FORK_NAME>.git (push)
+upstream        git@github.com:bigcommerce/catalyst.git (fetch)
+upstream        git@github.com:bigcommerce/catalyst.git (push)
 ```
 
-You can find documentation for each field in the `.env.local` file, described in [.env.example](https://github.com/bigcommerce/catalyst/tree/main/.env.example).
+### 3. Enable Corepack
 
-4. If you use VS Code, use the following command to configure VSCode with the project-specific settings the Catalyst team has created:
+Enable Corepack so that you can use `pnpm` as your Node.js package manager with Catalyst. When working in the monorepo, you must use `pnpm`, you cannot use `npm` or `yarn`.
 
 ```shell copy
-cp .vscode/settings.example.json .vscode/settings.json
+corepack enable pnpm
 ```
 
-5. Start the Catalyst development server!
+You can verify that the installation worked by running:
+
+```shell copy
+pnpm --version
+```
+
+> [!TIP]
+> You might see a confirmation prompt similar to `! Corepack is about to download https://registry.npmjs.org/pnpm/-/pnpm-X.X.X.tgz` and `? Do you want to continue? [Y/n]`. You should type `Y` to continue.
+
+If you see a version number printed to your console, you have successfully completed this step.
+
+### 4. Install Catalyst dependencies
+
+From the root of the monorepo, run the command below to install dependencies.
+
+```shell copy
+pnpm install
+```
+
+### 5. Set up environment variables
+
+```shell copy
+pnpm create @bigcommerce/catalyst@latest init
+```
+
+The CLI command above will prompt you to log into one of your BigCommerce stores, then create a new/use an existing Catalyst storefront Channel in your Control Panel’s Channel Manager section. It will then create an `.env.local` file with the required environment variables to run the Catalyst monorepository.
+
+### 6. (Optional) Copy VSCode settings
+
+If you use VS Code, use the following command to configure VSCode with the project-specific settings the Catalyst team has created:
 
 ```shell copy
-pnpm run dev
+cp .vscode/settings.example.json .vscode/settings.json
 ```
 
-The `dev` script runs all packages and apps in watch mode.
+### 7. Run the development server!
+
+Finally, you can run the following command from the monorepo root to run the development server:
+
+```shell copy
+pnpm dev
```

Comment:
Let's include `run` as part of the commands as tons of external documentation use this. Shorthand is great but coming from `npm`, a lot of people might not know about it.
```suggestion
pnpm run dev
```

Reply:
Great point!